### DISCUSSION TITLE
Using the regex manager to capture multiple dependencies in a single line

### DISCUSSION THREAD

**Discussion by jamietanna** [renovatebot/renovate#19461] [Request Help] [answered]
Is it possible to use [the regex manager](https://docs.renovatebot.com/modules/manager/regex/) to match multiple times? I.e. for https://stackoverflow.com/a/23214334 can you extract all the Gems and their versions in a single pattern?

**Comment by rarkins** [comment, marked as answer]
Try matchStringsStrategy=recursive

Outer one can match the gem install line, inner one can match each separate package:version
